src/Control/Session.php

```diff
@@ -586,13 +586,11 @@ public function save(HTTPRequest $request)
      * Recursively apply the changes represented in $data to $dest.
      * Used to update $_SESSION
      *
-     * @deprecated 4.1.0 Use recursivelyApplyChanges() instead
```

Comment:
Undeprecating because recursivelyApplyChanges() has different API - have raised [issue](https://github.com/silverstripe/silverstripe-framework/issues/10571) to investigate